library/std/src/io/mod.rs

```diff
@@ -1070,13 +1097,13 @@ impl<'a> IoSliceMut<'a> {
     /// ][..];
     ///
     /// // Mark 10 bytes as read.
-    /// bufs = IoSliceMut::advance(bufs, 10);
+    /// IoSliceMut::advance_slice(&mut bufs, 10);
     /// assert_eq!(bufs[0].deref(), [2; 14].as_ref());
     /// assert_eq!(bufs[1].deref(), [3; 8].as_ref());
     /// ```
     #[unstable(feature = "io_slice_advance", issue = "62726")]
     #[inline]
-    pub fn advance<'b>(bufs: &'b mut [IoSliceMut<'a>], n: usize) -> &'b mut [IoSliceMut<'a>] {
+    pub fn advance_slice(bufs: &mut &mut [IoSliceMut<'a>], n: usize) {
```

Comment:
Since "slice" is already part of the name of the type, `advance_slice` is a little ambiguous (does it mean advance inside the slice, or advance in a slice of slices). That could probably be improved just adding an 's' on the end, to make it `advance_slices`.

Reply:
Good point, will fix.